p2p/discovery/bucket_test.go

```diff
@@ -40,9 +41,13 @@ func Test_Bucket(t *testing.T) {
 }
 
 func getNode(port string) *Node {
-	id := getRandomNodeID()
+	id, err := common.GenerateRandomAddress()
+	if err != nil {
+		panic(err)
+	}
+
 	addr, _ := net.ResolveUDPAddr("udp", ":"+port)
```

Comment:
nice to extract a common to represent this addr combination logic:
"udp", ":"+port

Reply:
this will be temp, we will add ip part later